rrmngmnt/executor.py

```diff
@@ -101,5 +101,5 @@ def run_cmd(self, cmd, input_=None):
 
 
 class ExecutorFactory(object):
-    def build(self, host, user):
+    def build(self, host, user, disabled_algorithms):
```

Comment:
I agree with Meni, this parameter is specific to the ssh connection.

it should be added in rrmngmnt/ssh.py to the RemoteExecutorFactory class